remove references to Marketplace and High Fidelity (#162)
vircadia/vircadia-docs-sphinx: docs/source/host/maintain-domain/view-logs.rst

#######################
View Domain Server Logs
#######################

> Your domain server logs list the different events relevant to your High Fidelity domain. These can help you troubleshoot any issues with your domain and understand how the different parts of your domain communicate with each other.

Reviewer: ```suggestion
Your domain server logs list the different events relevant to your Vircadia domain. These can help you troubleshoot any issues with your domain and understand how the different parts of your domain communicate with each other.
```

Server logs are stored directly on the server and can be accessed as follows:

* **Windows**: Logs can be found in the directory "%AppData%\Local\Vircadia\Server Console\logs"

> * **Linux**: Logs can be accessed by running the command "journalctl -u vircadia-domain-server@default.service" for the domain server, or "journalctl -u vircadia-assignment-client@default.service" for the assignment client.

Reviewer: ```suggestion
* **Linux**: Logs can be accessed by running the command `journalctl -u vircadia-domain-server@default.service` for the domain server, or `journalctl -u vircadia-assignment-client@default.service` for the assignment client.
```

Reviewer 2: I don't think you know what you are doing here. ` writes stuff cursive. Double `` puts stuff it in a box.

Reviewer: I had originally done this with :code:`insert something here` and it was giving me odd results. Trying again it appears to work.

Author: should i close this one and commit something similar, but with \`these\` instead of \`\`these?\`\`

Reviewer: ``insert code here`` worked for me when testing so go ahead and try that locally. If it works, then it might be easiest to do it that way.

Author: ok, will commit then un-commit if it doesn't work

Author: it looks good in github, but i'm not sure how it looks after sphinx exports it.

Author: oh wait i read @JulianGro 's thing backwards.  never mind.

#######################
View Domain Server Logs
#######################

Your domain server logs list the different events relevant to your High Fidelity domain. These can help you troubleshoot any issues with your domain and understand how the different parts of your domain communicate with each other.

Server logs are stored directly on the server and can be accessed as follows:

> * **Windows**: Logs can be found in the directory "%AppData%/Local/Vircadia/Server Console/logs"

Reviewer 2: ```suggestion
* **Windows**: Logs can be found in the directory "%AppData%\Local\Vircadia\Server Console\logs"
```
Windows uses backslashes instead of slashes.

> * **Linux**: Logs can be found by running the command "journalctl -u vircadia-domain-server@default.service"

Reviewer 2: ```suggestion
* **Linux**: Logs can be accessed by running the command "journalctl -u vircadia-domain-server@default.service"
```
It doesn't show you where the logs are actually stored, so I would use different wording.

Also keep in mind that this will only show domain-server logs and not assignment-client logs. The assignment-clients do almost all the work. See https://docs.vircadia.com/explore/get-started/architecture.html#architecture-overview
